contracts/Market.sol

```diff
@@ -155,6 +160,12 @@ contract Market is
    */
   event PriorityRestrictedThresholdSet(uint256 threshold);
 
+  /**
+   * @notice Emitted on setting of `_priceMultiple`.
+   * @param priceMultiple The updated price multiple.
+   */
+  event SetPriceMultiple(uint256 priceMultiple);
```

Comment:
PriceMultipleSet?

Reply:
so Jaycen had asked me to change the tense of another event to match the active voice used in all the OZ dependencies (for example, `TransferBatch`, and now we have `SetHoldbackPercentage`)... what do you guys think?  I realize that it is not consistent with some of our other event names and personally I find it less intuitive in the active voice than in the past tense, but i'm fine with whatever approach

Reply:
Hmm, yeah I would rather have our names be internally consistent since these will be with us forever but not a big deal.

Reply:
Fwiw active voice seems to be the standard in most popular contracts. Same goes for documentation